packages/bson-bench/src/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { exists as existsCallback } from "fs";
+import { promisify } from "util";
+
+export const exists = promisify(existsCallback);
```

Comment:
```suggestion
export const exists = p => fs.access(p).then(() => true, () => false)
```

From FS promises you can use `access` and add a small chain to make it boolean. `exists` is deprecated.

Reply:
Funnily enough that was my initial implementation.